### ISSUE TITLE
Incorrect message in some edit boxes without suggestions[Bug]: 

### ISSUE THREAD

**Issue by Carlos-EstebanM** [nvdaes/controlUsageAssistant#19]
### Summary

In some editt boxes, the addon is saying that "After typing search text, press up or down arrow keys to review list of suggestions" when is not apropiate.

### To reproduce

1.	Open notepad in Windows 11, or any application with a standard edit box.
2.	Pay attention to the message announced by NVDA, with automatic messages or pressing NVDA + h.
NVDA says: “After typing search text, press up or down arrow keys to review list of suggestions”. This message is announced in all edit boxes, including standar editboxes without suggestions, like notepad, Biblos, etc.

### Expected behavior

Before of the learning message, NVDA check if the edit boxes show suggestions or not. When the condition is false, NVDA says a message similar to type text here. When the edit box has suggestions, like the search edit box in the start menu in Windows 10 or 11, NVDA says the correct message about suggestions with arrow keys.

### Relevant log output

```shell
IO - speech.speech.speak (23:54:38.463) - MainThread (19428):
Speaking ['Sin título: Bloc de notas']
DEBUG - NVDAObjects.NVDAObject._get_placeholder (23:54:38.667) - MainThread (19428):
Potential unimplemented child class: <NVDAObjects.Dynamic_EditableTextUseTextInfoToSpeakTypedWordsNVDAObjectExEditableTextWithAutoSelectDetectionEnhancedUIASupportUIAWindowNVDAObject object at 0x1370AE50>
IO - speech.speech.speak (23:54:38.673) - MainThread (19428):
Speaking ['Editor de texto', 'document', CancellableSpeech (still valid), 'blank']
IO - speech.speech.speak (23:54:38.675) - MainThread (19428):
Speaking [PitchCommand(), 'After typing search text, press up or down arrow keys to review list of suggestions.']
```

### Version

20231028

### System configuration

NVDA 2023.3.4, Windows 11 version is 23H2 (build 22631.3296)

**Comment by nvdaes** [comment]
Hello:
Thanks for reporting this. I'm investigating the best way to fix this bug, though seems that this is not easy, so this may take some time.

**Comment by nvdaes** [comment]
Fixed in #20
